src/NuGet/Microsoft.CodeAnalysis.EditorFeatures.Text.nuspec

```diff
@@ -14,9 +14,17 @@
       More details at https://aka.ms/roslyn-packages
 
       $commitPathMessage$
+      Microsoft.VisualStudio.CoreUtility: $MicrosoftVisualStudioCoreUtilityVersion$
```

Comment:
Let's say something more like "this depends on .... or higher". This is something package consumers see, and this is a library we expect extension authors to use.